instana/instrumentation/django/middleware.py

```diff
@@ -79,6 +95,37 @@ def process_exception(self, request, exception):
         if request.iscope is not None:
             request.iscope.span.log_exception(exception)
 
+    def __url_pattern_route(self, view_name):
+        from django.conf import settings
+        try:
+            from django.urls import RegexURLPattern as URLPattern
+            from django.urls import RegexURLResolver as URLResolver
+        except ImportError:
+            from django.urls import URLPattern, URLResolver
+
+        urlconf = __import__(settings.ROOT_URLCONF, {}, {}, [''])
+
+        def list_urls(lis, parent_pattern=None):
+            if parent_pattern is None:
+                parent_pattern = []
+            if not lis:
```

Comment:
We can put this condition in the very beginning, so if we see the incoming object to be empty, we can return